MODULE.bazel

```diff
@@ -262,15 +243,16 @@ http_archive(
     url = IU_251_URL,
 )
 
-CLION_242_SHA = "27029107088193b6ba0d3d888ce67fcdd1aae68f324c4c6ddadfdd4c3c90b87e"
+# The plugin api for intellij_ce_2025_2. This is required to build IJwB and run integration tests.
+IU_252_SHA = "f4c53631bbe1eef329d08195625578fbabf21a9a8580f607faef9c369e20a9d7"
 
-CLION_242_URL = "https://www.jetbrains.com/intellij-repository/releases/com/jetbrains/intellij/clion/clion/2024.2.4/clion-2024.2.4.zip"
+IU_252_URL = "https://www.jetbrains.com/intellij-repository/snapshots/com/jetbrains/intellij/idea/ideaIU/252.13776.59-EAP-SNAPSHOT/ideaIU-252.13776.59-EAP-SNAPSHOT.zip"
```

Comment:
I see CLion and IJ version mismatching, could you please just run `//tools/maintenance` with 252 only specified?